tensorflow/docs_src/install/install_java.md

```diff
@@ -181,7 +181,7 @@ Take the following steps to install TensorFlow for Java on Windows:
      [TensorFlow for Java on Windows](https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow_jni-cpu-windows-x86_64-1.8.0.zip).
   3. Extract this .zip file.
 
-
+__Note__: Please ensure that _MS Visual C++ 2015 Redistributable_ package is installed on Windows system as tensorflow JNI library (*tensorflow_jni.dll*) uses them at runtime.
```

Comment:
Thanks! A couple of questions

- Specifically Visual C++ 2015, or will later versions work?
- Visual C++ or Visual Studio?
- Is there a link that can be included to direct the user to what needs to be installed?

Perhaps rephrase as:

```
__Note__: The native library (`tensorflow_jni.dll`) requires `msvcp...dll` at runtime, which is included in the ...<linked text> package.
```

Reply:
* I checked with Visual C++ 2015 , Do you want me to check for later version as well?
* I installed Visual C++ 
* downloaded from here [Visual C++ 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)